static/scripts/rewards/render-transaction/render-transaction.ts

```diff
@@ -8,12 +8,18 @@ import { insertErc20PermitTableData, insertErc721PermitTableData } from "./inser
 import { renderEnsName } from "./render-ens-name";
 import { renderNftSymbol, renderTokenSymbol } from "./render-token-symbol";
 import { ERC20Permit, Permit, TokenType } from "@ubiquibot/permit-generation/types";
+import { displayCommitHash } from "./display-commit-hash";
 
 const carousel = document.getElementById("carousel") as Element;
+const footer = document.querySelector(".footer") as Element;
 const table = document.querySelector(`table`) as HTMLTableElement;
 type Success = boolean;
 
 export async function renderTransaction(): Promise<Success> {
+  displayCommitHash(); // @DEV: display commit hash in footer
+
+  footer.classList.add("animate");
```

Comment:
and this too.

The moving around is avoided with the animation alone and having it be called here means some paths it won't render at all.

Reply:
So basically just move the `deployCommitHash()` back to `renderTransaction()` where it was originally was

Reply:
Keep the function in it's own file, move the call and `classList.add` into `init`

Yes to where it was originally, my thoughts were "handle footer parallel to table rendering" but the animation alone should resolve the UI shift meaning the call didn't need to be moved that's on me.

Reply:
Alright got it! I've moved everything back.